Tutorials/Particles/ElectromagneticPIC/CMakeLists.txt

```
@@ -0,0 +1,60 @@
+#
+# Contains 3 tutorials
+#
+#  * CUDA
+#  * OpenMP
+#  * OpenACC (NOT supported anymore)
+#
+if ( (NOT ENABLE_CUDA AND NOT ENABLE_OMP) OR (NOT DIM EQUAL 3) )
+   return()
```

I wonder if we should write a little message such as
```suggestion
   message(STATUS "Tutorials - ElectromagneticPIC: Skipped (requires 3D with CUDA or OMP)")
   return()
```
instead of silently returning.

---

I was thinking the same. The problem is that most of the Tutorials have some conditionality attached to them so I will need to modify most of the relevant CMakeLists.txt and include a status message that depends on the name of the Tutorial itself. If the Tutorial name/path changes, the status message again will need to be modified accordingly. I am not sure this is worth the effort. What about just adding a disclaimer in the Docs?

---

Docs is a good idea as well, agreed.